src/olympia/reviewers/utils.py

```diff
@@ -867,6 +868,7 @@ def set_file(self, status, file):
         file.status = status
```

Comment:
Now that I'm looking at it again, this should probably have been a model method. `file.reset_disabled()`

Reply:
I'm talking about something different.

A new method on either the version or the file that essentially encapsulates all of the logic we have here in a utility (_recalling @diox  mentioning we should lean towards business logic in models_) into a method we can call for a particular file/version. 

(I guess the distinction is also somewhat opaque since previously it's been said there is little tangible difference between the two at this point)

No action needed unless @eviljeff agrees and is keen to implement but I thought it was worth mentioning there should be a way to add this on the model and that would probably be "better" in a latte art code kind of way.

Reply:
There's an argument we should have methods to reject a version, to approve a version, etc, but that's not how the reviewer tools currently works.  Something to consider for future refactoring though.